src/Detached.Annotations/NotAttachedAttribute.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Detached.Annotations
+{
+    [AttributeUsage(AttributeTargets.Property, Inherited = true, AllowMultiple = false)]
+    public sealed class NotAttachedAttribute : Attribute
```

Comment:
I discovered that [NotMapped] also unmaps fields from entity framework too
so I created [NotAttached] to fix this for now

Reply:
nothing to do with tests, just needed it for debugging

Reply:
Please comment intention in code. We didn't understand where or why to use.
So, it's just internal?!